code/game/gamemodes/setupgame.dm

```diff
@@ -49,6 +48,10 @@
 	GLOB.twitchblock        = getAssignedBlock("TWITCH",        numsToAssign)
 	GLOB.nervousblock       = getAssignedBlock("NERVOUS",       numsToAssign)
 	GLOB.wingdingsblock     = getAssignedBlock("WINGDINGS",     numsToAssign)
+	GLOB.mesonblock			= getAssignedBlock("MESONS",     numsToAssign, good=1)
+	GLOB.noflashblock		= getAssignedBlock("NOFLASH",     numsToAssign, DNA_HARDER_BOUNDS, good=1)
+	GLOB.nightvisionblock	= getAssignedBlock("NIGHTVISION",     numsToAssign, DNA_HARDER_BOUNDS, good=1)
```

Comment:
This whole file has spaces for indentation so probably fine to leave that, but may as well have them line up properly:
![image](https://user-images.githubusercontent.com/57483089/127350307-cdbafaca-346c-46fd-b402-6197f441b355.png)
![image](https://user-images.githubusercontent.com/57483089/127350412-7a6653c8-5866-4c96-b5de-ea7d8684818e.png)

```suggestion
	GLOB.mesonblock			= getAssignedBlock("MESONS",        numsToAssign, good=1)
	GLOB.noflashblock		= getAssignedBlock("NOFLASH",       numsToAssign, DNA_HARDER_BOUNDS, good=1)
	GLOB.nightvisionblock	= getAssignedBlock("NIGHTVISION",   numsToAssign, DNA_HARDER_BOUNDS, good=1)
```